translation-canary/translation_canary/translated/__init__.py

```diff
@@ -146,7 +146,7 @@ def testSourceTree(srcdir, releaseMode=False, modifyLinguas=True):
     success = True
     srcdir = os.path.normpath(srcdir)
 
-    for dirpath, _dirnames, paths in os.walk(srcdir):
+    for dirpath, _, paths in os.walk(srcdir):
```

Comment:
Our pylint config is set up to recognize that anything starting with an underscore is an unused variable.  I'd prefer to not have to go through and change all these.